cpp/src/io/comp/nvcomp_adapter.hpp

```diff
@@ -29,7 +29,7 @@
 
 namespace cudf::io::nvcomp {
 
-enum class compression_type { SNAPPY, ZSTD, DEFLATE };
+enum class compression_type { SNAPPY, ZSTD, DEFLATE, LZ4 };
```

Comment:
There's some conversation I overheard about LZ4 vs. LZ4 "raw". Is there a difference that we need to clarify in naming / comments somewhere? Are there any documents about the differences that developers should be aware of?

edit: Maybe answered here? https://github.com/rapidsai/cudf/pull/14906/files#r1473646421

Reply:
added a few comments to parquet code to help clarify.

Reply:
Looks great.